utils/download_glue_data.py

```diff
@@ -0,0 +1,141 @@
+''' Script for downloading all GLUE data.
```

Comment:
We need to add a link to the original location of this script.
And copyright notice as well.

Reply:
Fix pushed